src/modals/GeolocationModal.jsx

```diff
@@ -3,40 +3,50 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Modal from 'src/components/ui/Modal';
 import { listen } from 'src/libs/customEvents';
+import { Button, CloseButton } from 'src/components/ui';
+import classnames from 'classnames';
 
-const GeolocationModal = ({ status, onClose }) => {
+const GeolocationModal = ({ status, onClose, onClick }) => {
```

Comment:
`onClick` isn't a precise enough name at this level. It gives no idea of what it does or what kind of element it applies to. `onAccept` would be clearer.